app/controllers/assigned_applications_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
     )
   end
 
-  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength
+  # rubocop:disable Naming/AccessorMethodName, Metrics/MethodLength, Metrics/AbcSize
```

Comment:
This will need a refactor once the integration with the datastore is complete is complete. No point in worrying about it now but its definitely something to come back to to break down into smaller methods or add an abstraction to reduce the method size + cyclomatic complexity of it.